src/pages/admin/reports/components/program-expenses-section/components/program-expenses-table/ProgramExpensesTable.module.scss

```
@@ -54,9 +51,8 @@
 
 .th {
     @include type.subtitle-1-medium;
-    padding: 30px 20px;
+    padding: 16px 20px;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)